enterprisemediawiki/Meza1
# allow read&edit for requests from server (Parsoid)

This PR makes it so any requests originating from the server and going to the server are given read and edit rights on all wikis. This means that any services running on the server have the ability to read and edit wikis. The only current service requiring this is Parsoid. Without this Parsoid will not work on wikis which do not allow anonymous viewing and editing.

**How to test:**

1. After Meza1 install, perform normal MW tests
2. Modify `...htdocs/wikis/demo/config/setup.php` and add `$wgGroupPermissions['*']['read'] = false;` and `$wgGroupPermissions['*']['edit'] = false;`
3. Verify you cannot access the wiki with an anonymous user
4. Verify VE works with a logged in user

**Issues:**

* Closes #47
